packages/wmr/test/fixtures/publicpath-typescript/wmr.config.ts

```diff
@@ -0,0 +1,3 @@
+export default function (config) {
```

Comment:
I'm wondering if the `compileSingleModule` with `out='./cache'` will cause imports to be off by one directory level?

We could try doing a simple import of a second TS file here to check.